lib/vanagon/project/dsl.rb

```diff
@@ -159,6 +159,17 @@ def version_from_git
         warn "Directory '#{dirname}' cannot be versioned by git. Maybe it hasn't been tagged yet?"
       end
 
+      def version_from_branch
+        branch = Git.open(File.expand_path("..", Vanagon::Driver.configdir)).current_branch
+        if branch =~ /(\d+(\.\d+)+)/
+          return $1
+        else
+          warn "Can't find a version in your branch, make sure it matches \d+(\.\d+)+"
+        end
+      rescue Git::GitExecuteError => e
+        warn "Something went wrong trying to find your git branch.\n#{e}"
```

Comment:
Should this be a warning, or should this be a failure? Without this, won't artifacts become unversioned?